README.md

```diff
@@ -21,13 +21,13 @@ python setup.py develop
 ```
 
 ## [NEW] Tests
-You can use `tox` or `detox` to run the tests against multiple versions of python:
+You can use `tox` or `detox` to run the tests against Python 3.6 and 3.7:
 ```shell
 sudo dnf install python-detox
 detox
 ```
-Test outputs are stored under the [htmlcov-py27](htmlcov-py27) and [htmlcov-py36](htmlcov-py36) directory as 
-HTML files for Python versions 2.7 and 3.6 respectively. 
+Test outputs are stored under [htmlcov-py36](htmlcov-py36), [htmlcov-py37](htmlcov-py37) directories as 
```

Comment:
Should I not create them in the tox file as well then? Otherwise, this will still hold true (after a user runs `tox` the output files will be created under that directory.

Reply:
yes, you are right we won't create the files.

Reply:
Sounds good, I've removed creating the test in the tox.ini and related files. @pbortlov